src/ontology/mondo-edit.obo

```diff
@@ -7142,8 +7142,11 @@ replaced_by: MONDO:0006244
 [Term]
 id: MONDO:0000618
 name: Her2-receptor negative breast cancer
-comment: Editor note: check why not in NCIT; note also that triple-negative should be classified here
 xref: DOID:0060080 {source="MONDO:equivalentTo"}
+xref: EFO:0009780 {source="MONDO:equivalentTo"}
+xref: NCIT:C168519 {source="MONDO:equivalentTo"}
+xref: SCTID:431396003 {source="MONDO:equivalentTo"}
```

Comment:
Note that [bioregistry prefers](https://bioregistry.io/registry/snomedct) a `SNOMEDCT` prefix, but elsewhere MONDO appears to use `SCTID` so I went with that.